docs/develop-resources/README.md

```diff
@@ -0,0 +1,25 @@
+# Developing the Direct Resource Guide 
+
+We are thrilled to introduce the new way to add a ConfigConnector resource: the direct resource guide!  
+
+In the past few months, we have made a tremendous amount of efforts to make adding a new resource (or a new field) much faster and more manageable, we have changed some key resource reconciliation processs to be more reliable and Kubernetes-native. What's more, we have made a revolutionary change to the test driven development and PR review process to improve the test coverage for every single field. 
+
+There are definitely more work need to be done, but we'd like to share the steps about adding the direct resource so that it can benefit more users. We will conitnue improving this guide and making the step changes to make it simpler and easier to use. Please stay tuned for the upcoming changes.
```

Comment:
A few typos (e.g. conitnue), but I propose sending a PR "on top" of this one for efficiency...